### PR TITLE
Add and Remove methods

### PR DESCRIPTION
**Pull request by keroloussamy**
In this PR, I made the following
1- Add and remove method

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,59 +1,63 @@
...
+  });
+
+  const trashes = document.querySelectorAll('.fa-trash');
+  [...trashes].forEach((trash) => {
+    trash.addEventListener('click', (event) => {
+      LSOperations.remove(event);
+      const oldList = document.querySelectorAll('.todoItem');
+      [...oldList].forEach((e) => e.remove());
+      renderUI();
+    });
```
Comment (mikethreels) [inline src/index.js:40]: - [x] great job on implementing the functionality 👍  however the remove function is not working when clicked. please make sure it works before submitting another code review request.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kero, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (keroloussamy) [reply src/index.js:40]: Done

**Review by rukundoeric — APPROVED** [review]
Hi @keroloussamy,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**